server/src/main/java/org/opensearch/indices/RemoteStoreSettings.java

```diff
@@ -246,4 +272,12 @@ public int getMaxRemoteTranslogReaders() {
     private void setMaxRemoteTranslogReaders(int maxRemoteTranslogReaders) {
         this.maxRemoteTranslogReaders = maxRemoteTranslogReaders;
     }
+
+    public TimeValue getPinnedTimestampsSchedulerInterval() {
+        return pinnedTimestampsSchedulerInterval;
+    }
+
+    public TimeValue getPinnedTimestampsLookbackInterval() {
```

Comment:
Thinking if these can be static methods as setting is not dynamic. This will save from injecting `RemoteStoreSettings`